content/operate/rs/references/rest-api/requests/actions/_index.md

```diff
@@ -41,7 +41,18 @@ GET /actions
 
 ### Response {#get-all-response}
 
-Returns a JSON array of [action objects]({{< relref "/operate/rs/references/rest-api/objects/action" >}}) and an array of [state-machine objects]({{< relref "/operate/rs/references/rest-api/objects/state-machine" >}}).
+Returns a JSON array of [action objects]({{< relref "/operate/rs/references/rest-api/objects/action" >}}), which represent tasks, and an array of [state-machine objects]({{< relref "/operate/rs/references/rest-api/objects/state-machine" >}}).
+
+| Field | Type/Value | Description |
+|-------|------------|-------------|
+| action_uid | string | The action's globally unique identifier |
+| name | string | Name of the running or failed state machine |
+| progress | float (range: 0-100) | Percent of completed steps for the action |
+| status | "pending"<br />"active"<br />"completed"<br />"failed" | The action's status |
+| node_uid | string | UID of the node where the operation runs (optional) |
+| object_name | string | The object that the action runs on (optional) |
+| state | string | The current state of the state machine (optional)  |
+| pending_ops | JSON object | List of operations that are waiting to run (optional) |
```

Comment:
@rrelledge Could you please get more details from Nikolay about the fields that are included in pending ops and about it's strcuture

Reply:
Yes, I will ask.